### ISSUE TITLE
TV episode list is behind in some shows and will not update.

### ISSUE THREAD

**Issue by jhill54** [scakemyer/plugin.video.quasar#872]
Many shows, i.e. Devil's Canyon, do not show the most current episode even though a torrent is available.  Most times, if you search exactly for example 'devil's canyon s01e04' in the main search area, it will retrieve the torrent; in this example it does not however.  The main TV area for a particular season is behind and does not show past episode 3, but 6 episodes are available.

Kodi 17.3
Quasar 0.9.78

**Comment by ShlomiD83** [comment]
did you try deleting quasar's cache folder? sometimes it helps..

**Comment by jhill54** [comment]
How do I go about doing this? I’m running it on an android box.  I have used the settings to clear the cache, is that what you’re referring to?

**Comment by ShlomiD83** [comment]
no. i mean deleting the entire cache folder which is located in the userdata\addon\plugin.video.quasar folder while kodi is off and then starting it and letting quasar reload its entire cache, it can take a few minutes but from my experience it helps when there are missing episodes.

**Comment by jhill54** [comment]
Is this on a rooted android box?  I’m not seeing where the cache folder is.

**Comment by ShlomiD83** [comment]
you need to install es file explorer and enable hidden files, then you'll be able to access that folder.
Android/data/org.xbmc.kodi/files/.kodi/userdata/

**Comment by jhill54** [comment]
Does the device have to be rooted?  If so, what do you recommend to root the device, thanks.

**Comment by ShlomiD83** [comment]
no root is necessary.

**Comment by jhill54** [comment]
I did this, but it did not work.  Could you try devil’s canyon?  There are 6 episodes out, and it’s still only showing 3, thanks.

Jason

**Comment by ShlomiD83** [comment]
TMDB is  showing only 3 episodes, that's why.

**Comment by jhill54** [comment]
Is there a way to use a different tracker for this, it’s happening a lot on many shows that I watch.

**Comment by ShlomiD83** [comment]
ive added the episodes to TMDB, it might take a while but they'll show up eventually. the problem is until you or someone else will add the next episodes manually the wont show up in Quasar.

**Comment by jhill54** [comment]
I’m seeing 6 episodes on their website: https://www.themoviedb.org/tv/72076-devil-s-canyon/season/1?language=en-US <https://www.themoviedb.org/tv/72076-devil-s-canyon/season/1?language=en-US>

**Comment by jhill54** [comment]
Rather: http://thetvdb.com/?tab=season&seriesid=329246&seasonid=713886&lid=7 <http://thetvdb.com/?tab=season&seriesid=329246&seasonid=713886&lid=7> still shows 6 episodes.

**Comment by jhill54** [comment]
Hi there, the episodes are showing up now - how did you manually add them again?

**Comment by ShlomiD83** [comment]
register to TMDB, go to the show's page and press edit.
you can manually add episodes, just copy the details from TVDB.
